Week3/homework/step2-1.js

```diff
@@ -1,9 +1,8 @@
 'use strict';
 
-function foo(func) {
-  // What to do here?
-  // Replace this comment and the next line with your code
-  console.log(func);
+function foo() {
+
+  console.log('Week3');
 }
```

Comment:
Hmm. Actually, Foo should accept a function and then invoke/call that function. Don't see that happening here.

Reply:
@grod220 I changed it.